disputer/disputer.js

```diff
@@ -71,37 +73,52 @@ class Disputer {
           at: "Disputer",
           message: "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute.",
           id: disputeableLiquidation.id,
-          sponsor: disputeableLiquidation.sponsor
-        });
-
-        Logger.debug({
```

Comment:
This debug log was specifically requested in a previous PR so that a full error message would be printed to the console for someone trying to debug.

Reply:
Are you saying we should make this a `Logger.debug` instead of a `Logger.error`? I thought Logger.error would be at least as verbose as Logger.debug

Reply:
I added this back

Reply:
We can toss the entire error into the error log since that will go to the console too. I just don't want to lose the literal error message. Either way is fine with me.

Reply:
So while exploring this I realized that `Errors` don't get printed when you add them to the log, so I added this PR: https://github.com/UMAprotocol/protocol/pull/1136